src/_includes/styles.html

```diff
@@ -10,3 +10,4 @@
       integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH"
       crossorigin="anonymous" />
 <link rel="stylesheet" href="/stylesheets/main.css" />
+{% if page.stylesheet %}<link rel="stylesheet" href="/stylesheets/{{ page.stylesheet }}" />{% endif %}
```

Comment:
in a future iteration of this we can make `main.css` itself optional and load `dsdl.css` selectively here too 😎